sklearn/metrics/_plot/roc_curve.py

```diff
@@ -368,7 +423,7 @@ def from_predictions(
             error will be raised.
 
         name : str, default=None
-            Name of ROC curve for labeling. If `None`, name will be set to
+            Name of ROC curve for legend labeling. If `None`, name will be set to
             `"Classifier"`.
```

Comment:
I am wondering it this is still a good default, to name things "Classifier". Not sure if it provides more info, the legend label could just have the AUC value (or no legend)

Reply:
I would not change the default behaviour but I agree that maybe we would not need any legend.